tests/test_filter_count.py

```diff
@@ -341,6 +341,7 @@ def test_find_label_issues_multi_label(multi_label, filter_by):
         pred_probs=data["pred_probs"],
         filter_by=filter_by,
         multi_label=multi_label,
+        allow_missing_classes=multi_label,
     )
     acc = np.mean((data["labels"] != data["true_labels_train"]) == noise_idx)
```

Comment:
Are you sure this line is working properly in multilabel settings? Just want to confirm because I thought `data[labels]` is a list of lists in multilabel settings for which I'm not sure this would work properly?

Reply:
it creates a list of labels with all 1, which throws an error

Reply:
Right you should specially create properly formatted data for testing multilabel. The given & true labels need to be a list of lists, so I don't think you can simply assess the accuracy in this way.

Reply:
Got it, with this I can remove allow_missing_classes